src/app/newSettings/Sections/Workspace/Members/MembersSection.tsx

```diff
@@ -25,105 +28,35 @@ const MembersSection = ({ onClosePreferences }: { onClosePreferences: () => void
-  //     ],
-  //     storage: 2200000000000,
-  //     isActivityEnabled: true,
-  //     activity: [
-  //       {
-  //         date: 'Feb 13, 2024',
-  //         records: [],
-  //       },
-  //       {
-  //         date: 'Feb 12, 2024',
-  //         records: [
-  //           { title: 'Logged out', description: 'IP: 111.222.333', time: '12:35' },
-  //           {
-  //             title: 'Uploaded a file',
-  //             description: 'Drive/Marketing Team/January Campaign/Budget.pdf',
-  //             time: '12:35',
-  //           },
-  //           {
-  //             title: 'Created new folder',
-  //             description: 'Drive/Marketing Team/January Campaign',
-  //             time: '12:35',
-  //           },
-  //           {
-  //             title: 'Logged in',
-  //             description: 'IP: 111.222.333',
-  //             time: '12:35',
-  //           },
-  //         ],
-  //       },
-  //     ],
-  //     isTeams: true,
-  //     teams: [
-  //       { team: 'Deveolpment', role: 'owner' },
-  //       { team: 'Marketing', role: 'member' },
-  //     ],
-  //   },
-  //   {
-  //     id: '123124',
-  //     name: 'Michael',
-  //     lastname: 'Dalesom',
-  //     email: 'michael@internxt.com',
-  //     role: 'manager' as MemberRole,
-  //     products: [
-  //       { name: 'Drive', usageInBytes: 524000000000, color: 'primary' },
-  //       { name: 'Backups', usageInBytes: 12300000, color: 'indigo' },
-  //     ],
-  //     storage: 1100000000000,
-  //     isActivityEnabled: false,
-  //     activity: [
-  //       {
-  //         date: 'Feb 13, 2024',
-  //         records: [],
-  //       },
-  //     ],
-  //     isTeams: false,
-  //     teams: [],
-  //   },
-  //   {
-  //     id: '123123214',
-  //     name: 'Bea',
-  //     lastname: 'Donell',
-  //     email: 'bea@internxt.com',
-  //     role: '' as MemberRole,
-  //     products: [{ name: 'Drive', usageInBytes: 824000000000, color: 'primary' }],
-  //     storage: 1100000000000,
-  //     isActivityEnabled: true,
-  //     activity: [
-  //       {
-  //         date: 'Feb 13, 2024',
-  //         records: [],
-  //       },
-  //     ],
-  //     isTeams: true,
-  //     teams: [],
-  //   },
-  // ] as Member[];
+  const [guestsUsers, setGuestUsers] = useState<number>(0);
 
   useEffect(() => {
     const selectedWorkspaceId = selectedWorkspace?.workspace.id;
     getWorkspacesMembers(selectedWorkspaceId);
+    getWorkspacePendingInvitations(selectedWorkspace?.workspaceUser.workspaceId);
   }, []);
 
   useEffect(() => {
     const newMembers = searchMembers(members, searchedMemberName);
     setDisplayedMembers(newMembers);
   }, [searchedMemberName]);
 
+  const getWorkspacePendingInvitations = async (workspaceId) => {
+    try {
+      const workspaceGuestsUsers = await workspacesService.getWorkspacePendingInvitations(
+        workspaceId,
+        PENDING_INVITATIONS_LIMIT,
+        PENDING_INVITATIONS_OFFSET,
+      );
```

Comment:
This only gonna show 25 invitations, is the desired behaviour?

Reply:
At the moment you can only have 5 pending invitations but as the endpoint limit is 25 I have set it like this